File: app/src/main/java/opencrowd/hgc/hgcwallet/ui/onboard/EmptyScreen.java
package opencrowd.hgc.hgcwallet.ui.onboard;

import android.content.Context;
import android.support.annotation.NonNull;
import android.widget.Button;

import com.wealthfront.magellan.BaseScreenView;
import com.wealthfront.magellan.Screen;

import opencrowd.hgc.hgcwallet.R;


public class EmptyScreen extends Screen<EmptyScreenView> {

    @NonNull
    @Override
    protected EmptyScreenView createView(@NonNull Context context) {
        return new EmptyScreenView(context);
    }
}


class EmptyScreenView extends BaseScreenView<EmptyScreen> {
    Button mTermAccept;
    public EmptyScreenView(@NonNull Context context) {
        super(context);
        inflate(context, R.layout.view_empty,this);

    }

}

File: app/src/main/java/hedera/hgc/hgcwallet/ui/onboard/EmptyScreen.kt
package hedera.hgc.hgcwallet.ui.onboard

import android.content.Context
import android.view.View
import com.wealthfront.magellan.BaseScreenView
import com.wealthfront.magellan.Screen
import hedera.hgc.hgcwallet.R

class EmptyScreen : Screen<EmptyView>() {

    override fun createView(context: Context): EmptyView {
        return EmptyView(context)
    }
}

class EmptyView(context: Context) : BaseScreenView<EmptyScreen>(context) {
    init {
        View.inflate(context, R.layout.view_empty, this)
    }
}